src/otx/algo/action_classification/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Module for OTX action classification models, hooks, utils, etc."""
```

Comment:
is hooks still used in this version?

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2776/commits/669d935209ac14f857b52b25ce3957fc9ce03f15